src/plugins/data/public/ui/search_bar/lib/use_dataset_manager.ts

```diff
@@ -6,34 +6,34 @@
 import { useState, useEffect } from 'react';
 import { Subscription } from 'rxjs';
 import { Dataset } from '../../../../../data/common';
-import { DataSetContract } from '../../../query';
+import { DatasetContract } from '../../../query';
 
-interface UseDataSetManagerProps {
-  dataSet?: Dataset;
-  dataSetManager: DataSetContract;
+interface UseDatasetManagerProps {
+  dataset?: Dataset;
+  datasetManager: DatasetContract;
 }
 
-export const useDataSetManager = (props: UseDataSetManagerProps) => {
-  const [dataSet, setDataSet] = useState<Dataset | undefined>(
-    props.dataSet || props.dataSetManager.getDataSet()
+export const useDatasetManager = (props: UseDatasetManagerProps) => {
```

Comment:
```suggestion
export const useDataset = (props: UseDatasetManagerProps) => {
```

This is more accurate since thats how react hooks are usually named